### DISCUSSION TITLE
文本中中文括号应该怎么输入呢？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#104] [Q&A]
你好，我在文本中需要输入中文括号，当我直接输入，发现括号前的间距挺大的，至少明显比文献引用自动产生的括号间距大不少。如果采用数学的括号，似乎有点不合适。那么中文括号改怎么输入呢？

**Comment by tanukihee** [comment]
（）

**Comment by tanukihee** [comment]
> 至少明显比文献引用自动产生的括号间距大不少

自动生成的括号是西文括号，但正常行文还是要用中文括号的

**Reply by earthmoon** [reply]
也就是直接按键盘输入就可以对吧。

**Comment by sikouhjw** [comment]
文献引用的括号是英文括号 `()`，跟你输入中文括号、中文括号大不大没有一点关系。

如果你觉得间距太大了，请阅读 `texdoc xeCJK` 或者 `texdoc ctex`，用『开明式』或者『半角式』风格。

**Reply by earthmoon** [reply]
大就大吧，目前我完全没有修改的能力，能把你们做好的用好都需要慢慢学。
